src/components/editor/EditorArticleController.jsx

```diff
@@ -35,6 +35,7 @@ export default class EditorArticleController extends FalcorController {
         // As changedFlag is currently false, we can simply use equals here
         changedFlag = fields.some((field) => {
           let formValue = formNode[field].value;
+          console.log(typeof formValue);
           if (field === "issue") {
             formValue = parseInt(formValue);
```

Comment:
I believe you can give `<option>` tags a different value than what they display. So can avoid the parseInt in that way.